pkg/e2e/autoscaler/autoscaler.go

```diff
@@ -0,0 +1,371 @@
+package infra
```

Comment:
infra? is this an accidental carry over?
In any case is there any reason to have operator/autoscalers/infra/actuators.
I think they could all belong to same package e.g "elastic compute" and then have independent files and a common utils

Reply:
> infra? is this an accidental carry over?
Right, it should be `autoscaler`.

> In any case is there any reason to have operator/autoscalers/infra/actuators.
I think they could all belong to same package e.g "elastic compute" and then have independent files and a common utils

We are not exposing new API so in this case I prefer to allocate one directory/package for each important component. I still prefer smaller packages with few files rather than one package with a lots of files.